mobile/src/events/LocationPicker.tsx

```diff
@@ -0,0 +1,60 @@
+import React from "react";
+import { useField, useFormikContext } from "formik";
+import { GooglePlacesAutocomplete } from "react-native-google-places-autocomplete";
+
+// // FIGURE OUT HOW TO RESTRICT THIS // encode it 
+const api_key: string = "AIzaSyAYRQIS7VyzXVVlrcsch2SQO5qtW3dGOV4";
```

Comment:
We should try to keep api keys out of github - look at using environment variables to keep this stored in local files. So google smth like "How to use environment variables api key" or something like that.